ci/asan_tests/Makefile

```diff
@@ -0,0 +1,28 @@
+SHELL=/bin/bash
```

Comment:
We shouldn't use a Makefile for this by the way; it should be a script of some sort. (It also adds an unnecessary Make dependency.) We can keep it for now if it's time-sensitive, but let's turn it into a script later.